casbin/rbac/default_role_manager/role_manager.py

```diff
@@ -26,98 +26,145 @@ def add_domain_matching_func(self, fn=None):
         self.has_domain_pattern = True
         self.domain_matching_func = fn
 
-    def has_role(self, name):
-        if self.matching_func is None:
-            return name in self.all_roles.keys()
+    def has_role(self, role):
+
```

Comment:
Please remove `has_role()` `create_role()`，refer to [RoleManager](https://github.com/casbin/casbin/blob/aeeb9e6854e26bdbe22145e212b92c930df3036d/rbac/default-role-manager/role_manager.go#L33) .